<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerForm</class>
 <widget class="QWidget" name="customerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library: Customer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <widget class="QWidget" name="booksTab">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <attribute name="title">
       <string>Books</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="booksTable">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Genres</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Page count</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Author</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Publisher</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Count</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="orderBtn">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Order</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Filters</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item alignment="Qt::AlignHCenter">
                 <widget class="QLineEdit" name="lineEdit">
                  <property name="placeholderText">
                   <string>Book name</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignHCenter">
                 <widget class="QLineEdit" name="lineEdit_2">
                  <property name="placeholderText">
                   <string>Genre</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Author name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="authorsList">
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <item>
                   <property name="text">
                    <string/>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Publisher</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="publishersList">
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                  <item>
                   <property name="text">
                    <string/>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="searchBtn">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>Search</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_18">
                  <property name="text">
                   <string>Reset</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ordersTab">
      <attribute name="title">
       <string>My orders</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTableWidget" name="ordersTable">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Book name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Order date</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_16" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QWidget" name="widget_17" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QPushButton" name="cancelOrderBtn">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_18" native="true"/>
          </item>
          <item>
           <widget class="QWidget" name="widget_19" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
